store/src/java/com/zimbra/cs/account/ProvUtil.java

```diff
@@ -857,14 +857,24 @@ public void printHelp() {
             "{domain} {ouName} {newName}", Category.HAB , 3, 3),
         DELETE_HAB_OU("deleteHABOrgUnit", "dhou",
             "{domain} {ouName}", Category.HAB , 2, 2),
-        CREATE_HAB_GROUP("createHabGroup", "chabg",
+        CREATE_HAB_GROUP("createHABGroup", "chg",
             "{groupName} {ouName} {name@domain} {TRUE|FALSE} [attr1 value1 [attr2 value2...]]", Category.HAB , 3, Integer.MAX_VALUE),
-        GET_HAB("getHab", "ghab",
-            "{habRootGrpId} ", Category.HAB , 1, 1),
-        MODIFY_HAB_GROUP("modifyHabGroup", "mhab",
-            "{habGrpId} {habParentGrpId} {targetHabParentGrpId} ", Category.HAB , 3, 3),
-        MODIFY_HAB_GROUP_SENIORITY("modifyHabGroupSeniority", "mhsi",
-            "{habGrpId} {seniorityIndex} ", Category.HAB, 2, 2);
+        GET_HAB("getHAB", "ghab",
+            "{habRootGrpId}", Category.HAB, 1, 1),
+        MODIFY_HAB_GROUP("modifyHABGroup", "mhg",
```

Comment:
Wondering if wee should change at least fro the command from modify to move

Reply:
changed to moveHABGroup